### ISSUE TITLE
build fails when running from monorepo with hoisted dependencies

### ISSUE THREAD

**Issue by diminutivesloop** [scullyio/scully#1177]
### Description
I'm trying to add Scully to a Lerna-based monorepo with all dependencies installed at the repo root. When I try to build the Scully project I get an error complaining that I started Scully outside the project folder. I've tried fixing this by using `npm link` to symlink `node_modules/@scullyio/scully` into the Scully project directory but I still get the same error.

#### 🔬 Minimal Reproduction
https://github.com/diminutivesloop/scully-lerna-demo

#### 💻Your Environment

**Angular Version:**

<pre><code>
Angular CLI: 9.1.12
Node: 12.18.4
OS: win32 x64

Angular: 9.1.12
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router
Ivy Workspace: <error>

Package                            Version
------------------------------------------------------------
@angular-devkit/architect          0.901.12
@angular-devkit/build-angular      0.901.12
@angular-devkit/build-ng-packagr   0.901.12
@angular-devkit/build-optimizer    0.901.12
@angular-devkit/build-webpack      0.901.12
@angular-devkit/core               9.1.12
@angular-devkit/schematics         9.1.12
@ngtools/webpack                   9.1.12
@schematics/angular                9.1.12
@schematics/update                 0.901.12
ng-packagr                         9.1.5
rxjs                               6.6.3
typescript                         3.8.3
webpack                            4.42.0
</code></pre>

**Scully Version:**
<pre>
<code>
"@scullyio/init": "^1.0.1",
"@scullyio/ng-lib": "^1.0.0",
"@scullyio/scully": "^1.0.0",
</pre>
</code>
</pre>

## 🔥 Exception or Error

<pre><code>
you started scully outside of a scully project-folder,
or didn't install packages in this folder.
We can't find your local copy to start.
This can also happen on windows with PowerShell and mixed case path-names
</code></pre>

**Comment by jorgeucano** [comment]
@diminutivesloop we don't have support for Lerna, so I download your project and test it, I just add 
```
"devDependencies": {
    "@scullyio/init": "^1.0.1",
    "@scullyio/ng-lib": "^1.0.0",
    "@scullyio/scully": "^1.0.0"
  },
```
Into the project (scully-demo) and work, I suppose work with peerDependency too... But I'm not sure.

**Comment by diminutivesloop** [comment]
Ok I was able to work around the issue by running `npm install` in the scully packages subfolder. I do still think Scully should support running with hoisted dependencies as that project structure isn't unique to Lerna and most other tools I've used support it (e.g. Angular CLI, Gulp, Storybook). However I understand if that functionality isn't high priority right now. I may also try pnpm as an alternative monorepo-friendly package manager which might play nicer with Scully.
